src/coreclr/vm/ceeload.cpp

```diff
@@ -478,14 +478,23 @@ void Module::Initialize(AllocMemTracker *pamTracker, LPCWSTR szName)
     m_dwTypeCount = 0;
     m_dwExportedTypeCount = 0;
     m_dwCustomAttributeCount = 0;
+#ifdef PROFILING_SUPPORTED
+    // set profiler related JIT flags
+    if (CORProfilerDisableInlining()){
+        m_dwTransientFlags |= PROF_DISABLE_INLINING;
+    }
+    if (CORProfilerDisableOptimizations()){
+        m_dwTransientFlags |= PROF_DISABLE_OPTIMIZATIONS;
+    }
```

Comment:
```suggestion
    if (CORProfilerDisableInlining())
    {
        m_dwTransientFlags |= PROF_DISABLE_INLINING;
    }
    if (CORProfilerDisableOptimizations())
    {
        m_dwTransientFlags |= PROF_DISABLE_OPTIMIZATIONS;
    }
```
Follow the style used in the rest of the file